src/server/routes/store.js

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+import { json } from 'body-parser';
+
+import {
+  getSnapDetails
+} from '../handlers/store';
+
+const router = Router();
+
+router.use('/github/user', json());
+router.get('/snaps/details/:name', getSnapDetails);
```

Comment:
The `router.use` parameter doesn't match `router.get` here.  (We should possibly switch to chained calls for this kind of thing so that it's harder to get wrong.)